src/engine/GroupBy.cpp

```diff
@@ -1007,6 +1008,7 @@ GroupBy::isSupportedAggregate(sparqlExpression::SparqlExpression* expr) {
 
   if (dynamic_cast<AvgExpression*>(expr)) return H{AVG};
   if (dynamic_cast<CountExpression*>(expr)) return H{COUNT};
+  if (dynamic_cast<CountStarExpression*>(expr)) return H{COUNT};
```

Comment:
At least add a comment here that we reuse the COUNT implementation which leaves some optimization potential on the table, because it has to check that the result is defined etc.
I was wondering "where is the CountStarAggregateData".